packages/proxy/schema/index.ts

```diff
@@ -49,10 +49,79 @@ export const modelParamToModelParam: {
   stream_options: null,
   parallel_tool_calls: null,
   response_format: null,
-  reasoning_effort: null,
+  reasoning_effort: "reasoning_effort",
   stop: null,
 };
 
+const effortToBudgetMultiplier = {
```

Comment:
can you link to the openrouter docs which use a similar mechanism?

Reply:
here's where they mention them heuristic: https://openrouter.ai/docs/use-cases/reasoning-tokens#reasoning-effort-level

google has an interesting approach: 
![image](https://github.com/user-attachments/assets/47fb6e2e-6244-4ba0-a546-88328271e6d0)